docs/docs/getting_started.md

```diff
@@ -1,8 +1,10 @@
 # Getting Started
```

Comment:
I know this does not render differently, but, going forward, can we leave a newline between the section header and the content? e.g.

```markdown
# Getting Started

Hello world.
```